src/SectionMeetings.tsx

```diff
@@ -0,0 +1,163 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React from 'react';
+import * as Moment from 'moment';
+import { IMeeting } from './types';
+
+interface Props {
+  meetings: Array<IMeeting>;
```

Comment:
Add a question mark after the `meetings`, i.e. `meetings?: Array<IMeeting>`. That indicates meetings could be null or undefined